.rubocop.yml

```diff
@@ -1,2 +1,5 @@
 Style/StringLiterals:
   EnforcedStyle: single_quotes
+
+Metrics/LineLength:
+  Max: 120
```

Comment:
I added this to master to cut down on PR comment noise from hound sooner than later.